gradle-baseline-java-config/resources/idea/intellij-java-palantir-style.xml

```diff
@@ -44,6 +44,9 @@
             </value>
           </option>
         </GroovyCodeStyleSettings>
+        <JavaCodeStyleSettings>
```

Comment:
"Changes style" above should really be "Changes intellij autoformatter". Will make the corresponding change to eclipse